### DISCUSSION TITLE
Sphere can't be scaled non uniformly.

### DISCUSSION THREAD

**Discussion by AndreaCatania** [jrouwe/JoltPhysics#1117] [General]
Hello,

I've noticed that it's not possible to scale a sphere non-uniformly. I have a sphere that is scaled like this: `(3, 0.2, 3)`, and the function `JPH::Shape::ShapeResult ScaledShapeRes = Shape->ScaleShape(Scale);` returns the sphere decorated with a scaled shape whose scale is `(2.66, 2.66, 2.66)`.

![image](https://github.com/jrouwe/JoltPhysics/assets/8342599/38525df4-86de-47b2-acdd-98773b111abc)

Is it possible to scale a sphere as I'm doing, or is the only way to use a convex shape instead?

**Comment by jrouwe** [comment]
Spheres indeed cannot be scaled non-uniformly. The only way is to use a convex hull.

**Reply by AndreaCatania** [reply]
Ok, thanks!

**Reply by jrouwe** [reply]
Note that I created some documentation on scaling today: https://jrouwe.github.io/JoltPhysics/#scaling-shapes

**Comment by AndreaCatania** [comment]
Perfect, thanks!
